sdk/appconfiguration/azure-data-appconfiguration/src/configuration_client.cpp

```diff
@@ -0,0 +1,2316 @@
+  auto const httpStatusCode = rawResponse->GetStatusCode();
+
+  if (httpStatusCode != Core::Http::HttpStatusCode::Ok)
+  {
+    throw Core::RequestFailedException(rawResponse);
+  }
+
+  GetKeyValueResult response{};
+  {
+    auto const& responseBody = rawResponse->GetBody();
+    if (responseBody.size() > 0)
+    {
+      auto const jsonRoot
+          = Core::Json::_internal::json::parse(responseBody.begin(), responseBody.end());
+
+      response.Key = jsonRoot["key"].get<std::string>();
+
+      if (jsonRoot.contains("label") && !jsonRoot["label"].is_null())
+      {
+        response.Label = jsonRoot["label"].get<std::string>();
+      }
+
+      if (jsonRoot.contains("content_type") && !jsonRoot["content_type"].is_null())
+      {
+        response.ContentType = jsonRoot["content_type"].get<std::string>();
+      }
+
+      if (jsonRoot.contains("value") && !jsonRoot["value"].is_null())
+      {
+        response.Value = jsonRoot["value"].get<std::string>();
+      }
+
+      if (jsonRoot.contains("last_modified") && !jsonRoot["last_modified"].is_null())
+      {
+        response.LastModified = jsonRoot["last_modified"].get<std::string>();
+      }
+
+      if (jsonRoot.contains("tags"))
+      {
+        response.Tags = std::map<std::string, std::string>{};
+
+        for (auto const& kv : jsonRoot["tags"].items())
+        {
+          std::string value{};
+          value = kv.value().get<std::string>();
+          response.Tags.Value().emplace(kv.key(), value);
+        }
+      }
+
+      if (jsonRoot.contains("locked") && !jsonRoot["locked"].is_null())
+      {
+        response.Locked = jsonRoot["locked"].is_string()
+            ? (jsonRoot["locked"].get<std::string>() == "true")
+            : jsonRoot["locked"].get<bool>();
+      }
+
+      if (jsonRoot.contains("etag") && !jsonRoot["etag"].is_null())
+      {
+        response.Etag = jsonRoot["etag"].get<std::string>();
+      }
+    }
+  }
+
+  response.XMsClientRequestId
+      = Core::Uuid::Parse(rawResponse->GetHeaders().at("x-ms-client-request-id"));
+  response.SyncToken = rawResponse->GetHeaders().at("Sync-Token");
+  response.ETag = rawResponse->GetHeaders().at("ETag");
+
+  return Response<GetKeyValueResult>(std::move(response), std::move(rawResponse));
+}
+
+Azure::Response<PutKeyValueResult> ConfigurationClient::PutKeyValue(
+    PutKeyValueRequestContentType const& contentType,
+    std::string const& key,
+    std::string const& accept,
+    PutKeyValueOptions const& options,
+    Core::Context const& context)
+{
+  auto url = m_url;
+  url.AppendPath("kv/");
+  url.AppendPath(!key.empty() ? Core::Url::Encode(key) : "null");
+
+  url.AppendQueryParameter("api-version", Core::Url::Encode(m_apiVersion));
+  url.AppendQueryParameter("label", Core::Url::Encode(options.Label));
+  std::string jsonBody;
+  {
+    auto jsonRoot = Core::Json::_internal::json::object();
+
+    // TODO: Add support for KeyValue Entity
+    /* jsonRoot["key"] = options.Entity.Key;
```

Comment:
Our coding guidelines strongly discourage (as in "MUST NOT") the use of /* */ to comment out code. 

Use #if 0 because it's less fragile.